ui/src/environment.tsx

```diff
@@ -0,0 +1,18 @@
+// Items here should be added to environment.js.template as well.
```

Comment:
Should this file exist?

Reply:
Removed comment, had copied from VWB-ui which may not be applicable here